stats.py

```diff
@@ -415,6 +418,48 @@ def get_comments(revision: DiffDb.Revision, sessions: Sessions) -> list[dict]:
     return comments
 
 
+def get_review_groups(sessions: Sessions) -> list[dict]:
+    """Returns a dict of group names with the members of each group"""
+    groups = []
+
+    # Get the project objects that end in '-reviewers'.
+    projects = (
+        sessions.projects.query(ProjectDb.Project)
+        .filter(ProjectDb.Project.name.endsWith("-reviewers"))
+        .all()
+    )
+
+    for project in projects:
+        # Get a list of members of this group
+        edge_query_result = (
+            sessions.projects.query(ProjectDb.Edges)
+            .filter(
+                ProjectDb.Edges.src == project.phid,
+                ProjectDb.Edges.type
+                == PhabricatorEdgeConstant.PROJECT_HAS_MEMBER.value,
+            )
+            .all()
+        )
+
+        # Get the PHID of each member (the destination on the edge).
+        member_phids = {edge.dst for edge in edge_query_result}
+
+        member_names = []
+        for phid in member_phids:
+            name = get_user_name(phid, sessions)
+            member_names.append(name)
+
+        groups.append(
+            {
+                "group_id": project.id,
+                "group_name": project.name,
+                "group_members": member_names,
```

Comment:
While working on the ETL, it was called out that being able to query by email address was a hard requirement for the ETL. We should probably have both a `group_usernames` and `group_emails` field in each row of the ETL. You can use `get_user_email` to get the usernames.